### DISCUSSION TITLE
Найцікавіші новини місяця: липень 2023

### DISCUSSION THREAD

**Discussion by malor** [shopokodu/community#27] [Теми для подкасту]
- IPv4 за гроші на AWS: https://www.infoq.com/news/2023/08/aws-ec2-public-ipv4/
- RIP Kevin Mitnick https://www.reddit.com/r/programming/comments/154t3wc/rip_kevin_mitnick/
- RIP Bram Moolenaar https://www.reddit.com/r/programming/comments/15ivsje/bram_moolenaar_creator_of_vim_has_died/
- Announcing OverflowAI https://stackoverflow.blog/2023/07/27/announcing-overflowai/
- I scraped 14M+ dev jobs for 17 months and here are the Most Demanded Programming Languages https://www.devjobsscanner.com/blog/top-8-most-demanded-programming-languages/
- Apple already shipped attestation on the web, and we barely noticed https://httptoolkit.com/blog/apple-private-access-tokens-attestation/
- Рік Linux на декстопах: https://www.reddit.com/r/linux/comments/14uu7cx/finally_3/
- Про дистрибутиви Linux: https://www.reddit.com/r/linux/comments/152l1nu/slackware_turns_30/

**Comment by dmytrodanilenkov** [comment]
головне щоб не AI
